cypress/support/pages/course/pageCourses.js

```diff
@@ -74,6 +74,8 @@ class Courses {
             .click()
         }
       })
+      cy.wait('@board_api')
```

Comment:
Don't know but out of curiosity, can we combine both calls into one wait: like this:

`cy.wait(['@getUsers', '@board_api', '@userPermissions_api'])` 

If not then its fine to me